ros/angel_msgs/srv/QueryImageSize.srv

```diff
@@ -0,0 +1,4 @@
+# Query the current image dimensions
```

Comment:
You establish in the code that the width and height may be `-1` if an image has not been seen yet. That should be documented here.